packages/constants/src/category.ts

```diff
@@ -0,0 +1,259 @@
+interface IKey {
+    id: string;
+    label: string;
+    value: string;
+    url: string;
+}
+
+interface ICategory {
+    [category: string]: IKey;
+}
+
+function Key(id: string, label: string, value: string, url: string): IKey {
+  return {id, label, value: typeof value === 'undefined' ? id : value, url};
```

Comment:
can the `id`/`url` be derived from the `label`/`value` using a snake case formatter? i'm thinking this might standardise things instead of having future contributors having to think of the `id`/`url`

Reply:
updated

Reply:
as discussed, we don't know what id is for, but things will break in development if it does affect anything in a serious manner anyways, so we'll be combining this